packages/edit-site/src/components/site-hub/index.js

```diff
@@ -121,21 +121,47 @@ export const SiteHubMobile = memo(
 		const history = useHistory();
 		const { navigate } = useContext( SidebarNavigationContext );
 
-		const { dashboardLink, homeUrl, siteTitle } = useSelect( ( select ) => {
+		const {
+			dashboardLink,
+			homeUrl,
+			siteTitle,
+			isClassicThemeWithStyleBookSupport,
+		} = useSelect( ( select ) => {
 			const { getSettings } = unlock( select( editSiteStore ) );
-			const { getEntityRecord } = select( coreStore );
+			const { getEntityRecord, getCurrentTheme } = select( coreStore );
 			const _site = getEntityRecord( 'root', 'site' );
+			const isBlockTheme = getCurrentTheme().currentTheme?.is_block_theme;
+			const supportsEditorStyles =
+				getCurrentTheme().theme_supports[ 'editor-styles' ];
```

Comment:
Nit: We could get `currentTheme` above and re-used the value.

Reply:
496e46dc0accaea267214724e7118a925f70cb3b

I also integrated `getSettings` selector.